### DISCUSSION TITLE
Preserve timestamp last modified

### DISCUSSION THREAD

**Discussion by sufyankhan-cell** [iterate-ch/cyberduck#14734] [Q&A]
I am unable to preserve last modified time when I transfer data from my windows server to s3 using cyberduck.

**Comment by tatze96** [comment]
Timestamp preservation for S3 is only supported using the *S3 (Timestamps)* connection profile. You can install the connection profile through the Cyberduck preferences *Profiles* tab.

For more information regarding modification date retention for S3, refer to our [wiki page](https://docs.cyberduck.io/protocols/s3/#modification-date).

**Reply by sufyankhan-cell** [reply]
After follow these steps i can see old date and time but in s3 still showing with new date & time is there any way or something i am missing here. please Respond ASAP

**Reply by tatze96** [reply]
Using the previously mentioned *S3 (Timestamp)* connection profile, the modification date will be preserved on upload through Cyberduck, but the creation date will be set to the current date. The creation and modification dates will be set to the current date if you access or edit and save the file.

We're writing the modification date into a custom metadata value that can be read out by Cyberduck and Mountain Duck. The S3 server itself doesn't use this custom metadata value and will therefore display the creation date.
